optimum/intel/openvino/utils.py

```diff
@@ -21,6 +21,7 @@
 from typing import Tuple, Type, Union
 
 import numpy as np
+import torch
```

Comment:
should we be more careful importing torch here? I suppose it may be not available

Reply:
`torch` is a requirement in optimum https://github.com/huggingface/optimum/blob/main/setup.py#L19